README.md

```diff
@@ -56,3 +58,14 @@ to reflect these differences.
 The code and our documentation are the ultimate sources of truth. CFPs are merely
 the starting point for the implementation.
 
+# Status
+
+ The status of a CFP indicates where it is in the process. The CFP process is as follows:
+
+ 1. ``Provisional``: All new PRs are given this status. This CFP's design has not yet been approved.
+ 2. ``Implementable``: Approved PRs that are merged are given this status. Their CFP design has been approved, and now the work can be started to implement them.
+ 3. ``Experimental``: The CFP has been implemented and is now in a 'Beta' state.
```

Comment:
Is it experimental, or is it beta? These words have different connotations ranging from "Use at your own risk" to "Please run this in production and report your findings".

I recognize the need for a category like this between "Implementable" and "Stable", though I'm not familiar enough with process or how the process impacts human behaviour in order to have a strong opinion on how to portray a proposal in this status.

I can see benefits in leaning it towards wording like "experimental" as a way to incentivize contributors to work on moving the CFP towards "Stable" state, though I could equally see contributors simply ignoring the CFP process after it becomes "Implementable" and completely focusing on stabilizing the functionality to the point where it's production-ready.

I'll CC @youngnick in here in case he has some opinion about how a process like this should look, as I think he has more experience than me on this.

Reply:
Related: For the Cilium project, this is likely to differ from other communities like Gateway API, because these designs in Cilium are targeted for typically one implementation in Cilium, and we have additional expectations about testing, docs, etc for that implementation as we accept changes. On the other hand when the entire project is a standard, then some of those requirements may end up in the design repo.